src/reports/RouteReportPage.jsx

```diff
@@ -162,6 +166,16 @@ const RouteReportPage = () => {
                       />
                     </TableCell>
                   ))}
+                  <TableCell style={{ paddingTop: '0px', paddingBottom: '0px' }}>
+                    <CollectionActions
+                      itemId={item.id}
+                      endpoint="positions"
+                      readonly={readonly}
+                      handleRemoveResult={() => {
```

Comment:
I think instead of adding `handleRemoveResult`, we can just reuse `setTimestamp`. It will only be called on deletion. A bit hacky and indirect, but doesn't require a completely new parameter.